src/build.py

```diff
@@ -234,6 +241,26 @@ def Tar(directory, print_content=False):
   return tar
 
 
+def Zip(directory, print_content=False):
+  assert os.path.isdir(directory), 'Must be a directory'
+  (dir, basename) = os.path.split(directory)
+  archive = os.path.join(dir, basename + '.zip')
+  print 'Creating zip archive', archive
```

Comment:
Do we want to always print this?
Presumably yes because `Tar` does this via `check_call`